espnet2/layers/global_mvn.py

```diff
@@ -52,8 +52,18 @@ def __init__(
             var = sum_square_v / count - mean * mean
         std = np.sqrt(np.maximum(var, eps))
 
-        self.register_buffer("mean", torch.from_numpy(mean))
-        self.register_buffer("std", torch.from_numpy(std))
+        if isinstance(mean, np.ndarray):
```

Comment:
@simpleoier this we should check carefully if it is ok also in the other recipes (it should IMO)

Reply:
basically this was failing with WavLM when statistics are accumulated because mean was a tensor

Reply:
This is strange. We never met this problem before. Firstly, we usually do not use global mvn for WavLM / HuBERT. Secondly, the statistics collect in stage 10 are numpy arrays, converted [here]( https://github.com/espnet/espnet/blob/e37ee270a238fad6d38e5e7d37b459c6c64d55fc/espnet2/main_funcs/collect_stats.py#L80)

Reply:
I can reproduce the error however whenever I try to use WavLM with global_mvn. 
Any reason why the conversion does not happen ?

Reply:
I also face an error there, when I try to use WavLM with global_mvn.

Reply:
I tried it. I didn't have such problem if I use correct config. However, I got this error `TypeError: expected np.ndarray (got numpy.float64)` when I used global_mvn, while in the config, I specified `extract_feats_in_collect_stats: False`. Did you change this one?

Reply:
Thank you for clarification. Actually, I specified `extract_feats_in_collect_stats: False`.

Reply:
I see. This may be the reason. If you specify it as False, it will return the speech (a 1D tensor) instead of feats (a 2D tensor).

Reply:
I understood. Thank you for your detailed follow-up.
In generally, it is recommend to use `utt_mvn` for HuBERT/WavLM, right?

Reply:
Yes. Utt_mvn is recommended.

Reply:
So what we do, should I remove this since now I use utterance mvn ?

Reply:
Yes. I think either we use global_mvn or utterance_mvn, we do not need this modification. The original code should work. It's probably a mistake in the config.

Reply:
Then we must raise an error to avoid this. 
any global_* normalization should have extract_feats_in_collect_stats: True right ?